.github/workflows/test.yml

```diff
@@ -236,21 +236,19 @@ jobs:
         with:
           name: pulumi-${{ runner.os }}-${{ runner.arch }}
           path: artifacts/go
-      - name: Install pulumi-language-python-exec
-        run: |
-          mkdir -p bin
-          cp sdk/python/cmd/pulumi-language-python-exec bin/
       - name: Install Pulumi Go Binaries
         run: |
+          echo "Running prep script to put artifacts in bin"
```

Comment:
A bit confused so wanted to clarify:

- this will install from sdk/python/cmd/pulumi-language-python-exec source to bin/ as before

- this will NOT use the "released" copy that got included in the goreleaser output during build.yml
   - because the scripts are not modeled as "binaries"

Reply:
Correct, but the change to GoReleaser here ensures this folder will be populated the same as the archives are prepared:


```yaml
archives:
  ...
  files:
    - src: bin/{{ .Os }}/*
```

As long as this is load bearing, we can't add a file to our release tarball without adding it to the tests PATH.